3.2.4 test failure

=================================== FAILURES ===================================
___________________ NonRegressionTests.test_numpy_distutils ____________________

self = <tests.test_regrtest.NonRegressionTests testMethod=test_numpy_distutils>

        @unittest.skipUnless(HAS_NUMPY, "Needs numpy")
        def test_numpy_distutils(self):
            """Special handling of virtualenv's patching of distutils shouldn't interfere
            with numpy.distutils.
    
            PY312_PLUS -- This test will likely become unnecessary when Python 3.12 is
            numpy's minimum version. (numpy.distutils will be removed then.)
            """
            node = extract_node(
                """
    from numpy.distutils.misc_util import is_sequence
    is_sequence("ABC") #@
    """
            )
            inferred = node.inferred()
>           self.assertIsInstance(inferred[0], nodes.Const)
E           AssertionError: Uninferable is not an instance of <class 'astroid.nodes.node_classes.Const'>

tests/test_regrtest.py:118: AssertionError
=========================== short test summary info ============================
FAILED tests/test_regrtest.py::NonRegressionTests::test_numpy_distutils - Ass...
= 1 failed, 1629 passed, 40 skipped, 7 deselected, 16 xfailed in 69.16s (0:01:09) =

This is on Fedora, fails on Python 3.12 and 3.13.

--- a/tests/test_regrtest.py
+++ b/tests/test_regrtest.py
@@ -11,6 +11,7 @@ import pytest
 
 from astroid import MANAGER, Instance, bases, manager, nodes, parse, test_utils
 from astroid.builder import AstroidBuilder, _extract_single_node, extract_node
+from astroid.const import PY312_PLUS
 from astroid.context import InferenceContext
 from astroid.exceptions import InferenceError
 from astroid.raw_building import build_module
@@ -100,7 +101,7 @@ multiply([1, 2], [3, 4])
         inferred = callfunc.inferred()
         self.assertEqual(len(inferred), 1)
 
-    @unittest.skipUnless(HAS_NUMPY, "Needs numpy")
+    @unittest.skipUnless(HAS_NUMPY and not PY312_PLUS, "Needs numpy and < Python 3.12")
     def test_numpy_distutils(self):
         """Special handling of virtualenv's patching of distutils shouldn't interfere
         with numpy.distutils.

